playbooks/roles/common/tasks/wait-nodes.yml

```diff
@@ -2,9 +2,9 @@
 - name: "wait for server to respond on ssh port"
   local_action: wait_for
     host={{ ansible_ssh_host | default(inventory_hostname) }}
-    port=22
+    port={{ ansible_port | default(22) }}
     search_regex=OpenSSH
-    delay=10
+    delay=5
```

Comment:
Is this important for Vagrant? It was creating problems to have it lower than 10